lib/composers/launchModeComposer.js

```diff
@@ -0,0 +1,14 @@
+const {makeModifierComposer} = require('../utils/makeComposer');
+const composers = require('../constants/composer');
+const {validate, validators} = require('../validation');
+const {invalidInputMessage} = require('../texts');
+
+/**
+ * Defines launchMode method
+ */
+const launchModeComposer = makeModifierComposer(composers.LAUNCH_MODE, ['launchMode'], (_, meta, value) => ({
+	...meta,
+	sendText: validate(validators.STRING, value, invalidInputMessage('launchMode', 'value')),
```

Comment:
seems that `sendText` is incorrect propperty.
Also define validation for accepting only `resume` and `restart` values